bikeshare_app/views.py

```diff
@@ -107,6 +107,13 @@ def register():
         # Allow user through if check passed, else log attempt and return home
         return render_template('register.html')
 
+@app.route('/checkin', methods=['GET'])
```

Comment:
Yes, the routing function won't trigger unless the HTTP method matches

On 5/2/16, Adam Nieto <notifications@github.com> wrote:
>> @@ -107,6 +107,13 @@ def register():
>>          # Allow user through if check passed, else log attempt and return
>> home
>>          return render_template('register.html')
>>
>> +@app.route('/checkin', methods=['GET'])
>
> Should I remove the logic underneath the function
> `    if request.method == 'GET':
>         # Allow user through if check passed, else log attempt and return
> home
>         return render_template('checkin.html')`
> and instead just return the template?
> Also, should I remove the the `methods = ['GET]` parameter of the function.
>
>
> ---
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/BinghamtonCoRE/bikeshare/pull/56/files/d9ad135d8c9e841d38d42d2c5a3352c1b1fa149a#r61767702


-- 
// Taylor Foxhall